app/enterprise/2.3.x/vitals/vitals-influx-strategy.md

```diff
@@ -9,45 +9,106 @@ redirect_from:
 
 This document covers integrating Kong Vitals with a new or existing InfluxDB
 time series server or cluster. Leveraging a time series database for Vitals data
-can improve request and Vitals performance in very-high traffic Kong Enterprise
+can improve request and Vitals performance in very-high traffic {{site.ee_product_name}}
 clusters (such as environments handling tens or hundreds of thousands of
 requests per second), without placing addition write load on the database
 backing the Kong cluster.
 
 For using Vitals with a database as the backend (i.e. PostgreSQL, Cassandra), 
 please refer to [Kong Vitals](/enterprise/{{page.kong_version}}/admin-api/vitals/).
 
-## Getting Started
+## Setting up Kong Vitals with InfluxDB
 
-### Preparing InfluxDB
+### Step 1. Install {{site.base_gateway}}
```

Comment:
I thought Lena said not to use variables in headings?

Might be a best practice to avoid variables in headings? Not sure. For linking purposes without a custom id for instance.

Reply:
ah, ok, will fix